File: PHP/uberPool.php
<?php
require_once('car.php');

class UberPool extends Car{
    public $model; 
    public $brand;

    public function __construct($driver, $license, $brand, $model){
        parent::__construct($driver, $license);
        $this->brand = $brand;
        $this->model = $model;
    }
}

?>

File: Java/UberPool.java
public class UberPool extends Car {
    String brand;
    String model;
    private Integer passenger;

    public UberPool(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.model = model;
        this.brand = brand;
    }

    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 6) {
            this.passenger = passenger;
        } else {
            System.out.println("Error, passenger isn't correct to this type of car.");
        }
    }

    public Integer getPassenger() {
        return passenger;
    }

}
